source/includes/_resources.md

```diff
@@ -0,0 +1,303 @@
+description | `string`| Description of resource item
+image_url | `string` | Image associated with resource
+reference_id | `string` | Reference id associated with resource
+product | `object`  | [`Product`](#Products) associated with the resource
+status | `string` | Status for the resource. Default is `CREATED`. Additional: `PROCESSING`, `UTILIZED`, `IDLE`
+
+## Create a new resource
+Created a new resource associated with the  [`User`](#Users).
+
+> Definition
+
+```
+POST https://api.shareactor.io/resources
+```
+
+
+> Example request:
+
+``` http
+POST /resources HTTP/1.1
+Content-Type: application/json
+Authorization: Bearer <jwt>
+X-Share-Api-Key: <shareactor-api-key>
+Host: api.shareactor.io
+
+{
+    "product": "58f9f856b70e2a56c4a0db3d",
+    "description": "description of the resource",
+    "name": "name of resource",
+    "image_url": "http://docs.shareactor.io/images/logo.png"}
+```
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "updated": {"$date": 1500275774067},
+    "_id": {"$oid": "596c643ed57ba203be2cf1c9"},
+    "user": {"$oid": "57ee9c72d76d431f85111432"},
+    "description": "description of the resource",
+    "image_url": "http://docs.shareactor.io/images/logo.png",
+    "name": "name of resource",
+    "product": {"$oid": "58f9f856b70e2a56c4a0db3d"},
+    "status": "CREATED",
+    "deleted": false,
+    "created": {"$date": 1500275774067},
+    "company": {"$oid": "57ee9c71d76d431f8511142f"}
+}
+
+```
+
+
+Attribute | Type | Description
+--------- | ---- | -------
+name | `string` | Name of Resource item
+description | `string`| Description of resource item
+status | `string` | Status for the resource. Default is `CREATED`. Additional: `PROCESSING`, `UTILIZED`, `IDLE`
+product | `string`  | [`Product`](#Products) id to be associated to the resource
+image_url | `string` | Image url associated with resource
+reference_id | `string` | Customizable id for resource
+
+
+## Receive a Resource by id
+Receive a resource, based on its id.
+
+> Definition
+
+```
+GET https://api.shareactor.io/resources/<resource_id>
+```
+
+> Example request:
+
+``` http
+GET /resources/596c643ed57ba203be2cf1c9 HTTP/1.1
+Content-Type: application/json
+Authorization: Bearer <jwt>
+X-Share-Api-Key: <shareactor-api-key>
+Host: api.shareactor.io
+```
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "updated": {"$date": 1500275774067},
+    "_id": {"$oid": "596c643ed57ba203be2cf1c9"},
+    "user": {"$oid": "57ee9c72d76d431f85111432"},
+    "description": "description of the resource",
+    "image_url": "http://docs.shareactor.io/images/logo.png",
+    "name": "name of resource",
+    "product": {"$oid": "58f9f856b70e2a56c4a0db3d"},
+    "status": "CREATED",
+    "deleted": false,
+    "created": {"$date": 1500275774067},
+    "company": {"$oid": "57ee9c71d76d431f8511142f"}
+}
+
+```
+
+
+## Update resource
+Update attributes in resource
+
+Attribute | Type | Description
```

Comment:
`Attribute` should be `Arguments` no?

I mean when it's a function/method, the word is `Arguments` or `Parameters`.
Maybe I'm wrong.

What do you think @pedropregueiro  ?